contract/Makefile

```diff
@@ -6,9 +6,20 @@ BLD=000000ubld
 ATOM_DENOM=ibc/BA313C4A19DFBF943586C0387E6B11286F9E416B4DD27574E6909CABE0E342FA
 ATOM=000000$(ATOM_DENOM)
 
-# your deployer address made from make add-address
-# ADDR=agoric1k78s7qz7rxy8afyjrqk3dntg8m83zaw3upe60p
-ADDR=agoric125d8s0tamx50gpqdd3m7f5gtkzkcnjs0ehsujr
+
+# fetch the address of the deployer account from the chain - you may need to adjust this to match your setup
```

Comment:
Why would we need to adjust this? 
Isn't the point of this pr to take care of that?